### ISSUE TITLE
Upright meaning

### ISSUE THREAD

**Issue by cotwitch** [troydraws/paddy-color-theme#11]
Hi @yl92 ! 
Firstly, let me thank for your great work!

Could you, please, explain, what means `upright` ? 

I'm curious about what is it, but unfortunately, I didn't find it anywhere in the description or elsewhere... :( 
Kindly thank you beforehand for explanation.

Regards

**Comment by yl92** [comment]
Thank you, glad you like it! 
The theme started with a version that used italics for functions / keywords etc. and later I added a version for each option that does not use italics, which I called `upright`. Some fonts, like IBM Plex Mono, has quite distinct italic design, which was why I used them in the first place – but then some other fonts are not as nicely done for italics and looks better without.

Example with italics:
<img width="491" alt="image" src="https://user-images.githubusercontent.com/13817751/164650272-2812c8a6-ac56-46a7-834a-02f61b6146bc.png">
Example without:
<img width="480" alt="image" src="https://user-images.githubusercontent.com/13817751/164650332-ca5d7442-26a7-481a-b27f-b2e4eb80445e.png">

Probably a good idea to clarify it somewhere yeah!
